packages/orbit-components/src/Popover/index.js

```diff
@@ -1,52 +1,77 @@
 // @flow
 import * as React from "react";
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 
-import Portal from "../Portal";
-import PopoverContentWrapper from "./components/ContentWrapper";
-import type { Props } from "./index.js.flow";
-import useTheme from "../hooks/useTheme";
 import useStateWithTimeout from "../hooks/useStateWithTimeout";
-import { POSITIONS, ALIGNS } from "./consts";
+import useTheme from "../hooks/useTheme";
+import { PLACEMENTS } from "./consts";
+import PopoverContent from "./components/ContentWrapper";
+import mq from "../utils/mediaQuery";
+import Portal from "../Portal";
+import defaultTheme from "../defaultTheme";
 import handleKeyDown from "../utils/handleKeyDown";
 
+import type { Props } from ".";
+
 const StyledPopoverChild = styled.div`
   position: relative;
 `;
 
+const StyledPopoverContent = styled.div`
+  ${({ theme }) => css`
+    overflow: auto;
+    border-top-left-radius: 12px;
+    border-top-right-radius: 12px;
+    width: 100%;
+    background-color: ${theme.orbit.paletteWhite};
+  `}
+
+  ${mq.largeMobile(css`
+    border-radius: 3px;
+  `)}
+`;
+
+// $FlowFixMe: https://github.com/flow-typed/flow-typed/issues/3653#issuecomment-568539198
+StyledPopoverContent.defaultProps = {
+  theme: defaultTheme,
+};
+
```

Comment:
This component is unused. Too bad ESLint doesn't report it, I guess because of setting `defaultProps`.